learn/app-development/widgets/datalive/form/form-layouts.md

```diff
@@ -5,9 +5,10 @@ sidebar_label: "Layouts"
 ---
 ---
 
-Form **layout** can be _1-column_, _2-column_ or _3-column_. Apart from this, the **alignment of the caption** for the form fields can be configured to be placed at the _top_, _left_ or _right_ of the form fields.
+Form **layout** can be _1-column_, _2-column_ or _3-column_. Apart from this, the **alignment of the caption** for the form fields can be configured to be placed at the _top_, _left_, _right_ or _floating_ of the form fields.
 
 1. 1-Column Form with left-aligned Captions
 2. 2-Column Form with right-aligned Captions
 3. 3-Column Form with top-aligned Captions
+4. 3-Column Form with floating Captions (Angular material style)
```

Comment:
Link Angular material style to this https://material.angular.io/components/form-field/examples

Reply:
done